protos/sources/buf.yaml

```diff
@@ -7,6 +7,7 @@ lint:
   except:
     - PACKAGE_DIRECTORY_MATCH
     - PACKAGE_VERSION_SUFFIX
+    - RPC_REQUEST_RESPONSE_UNIQUE
```

Comment:
Are you sure we want to globally enable this? It can be done on a case-by-case basis with a comment

Reply:
@pscott31 Ah, I wasn't aware of that one. I'll give it a shot, thanks!